data/publiccloud/restart_instance.sh

```diff
@@ -71,6 +69,7 @@ esac
 wait_for_power_off "$HOST" "$CNT"
 wait_for_power_on "$HOST" "$CNT"
 echo "Instance $INSTANCE_ID restarted";
+sleep 5 # need pause for slow SUT to fully initialize sshd
```

Comment:
Would it make sense in run_ltp to use the many perl publiccloud routines for CSP management in place of deep dive in shell scripts that prevent the test call control that we see in autoinst.log ?

I.e. we have `lib/publiccloud/instance.pm::wait_for_ssh` widely used for starting stopping instances.

Reply:
this is needed because reboot and connection here is handled directly by Kirk and OQa itself has no idea about reboot in progress